vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -315,6 +315,40 @@ public Integer findMaxContentOrder(String slideId) {
         return contentBlockRepository.findMaxContentOrder(slideId);
     }
 
+    /**
+     * Adjusting the content order of the blocks of slide once it is dragged and changed position
+     * 
+     *  @param blockId - id of block whose content order needs to be adjusted.
+     *  @param contentOrder - value with which contentOrder of block needs to be updated.
+     *  
+     */
+
+    @Override
+    public void adjustContentOrder(List<ContentBlock> contentBlockList) throws BlockDoesNotExistException {
```

Comment:
let's call this `updateContentOrder` since you are really not adjusting anything, just updating the db with the given values.